lib/ImportStatement.js

```diff
@@ -301,7 +303,7 @@ export default class ImportStatement {
     }
   }
 
-  _sideEffectImportString(): string {
+  _assignmentLessImportString(): string {
```

Comment:
The naming of this method is ambiguous to me now. How about `_importStringWithoutAssignment`?